src/routes/PaymentPage/PaymentPage.tsx

```diff
@@ -0,0 +1,130 @@
+import { ReactElement, useState, ChangeEvent } from 'react';
+import './PaymentPage.css';
+import { DeliveryCard, RadioInput, Button } from '../../components';
+import { useSelector, useDispatch } from 'react-redux';
+import { RootState } from '../../rootReducer';
+import { useHistory } from 'react-router-dom';
+import { v4 as uniqueId } from 'uuid';
+import {
+	selectAddressById,
+	Address,
+	selectAllCartItems,
+	ProductsEntity,
+	CartItem,
+	Order,
+	addOrder,
+	emptyCart,
+	NewToastParams,
+	addNewToast,
+} from '../../slices';
+
+const getMonth = (monthNum: number): string => {
```

Comment:
put this function utils file. this is just a simple utility.
 this code can be very short if you just have an array with the name of months and return month based on Index 😄